_posts/2022-08-24-cockpit-275.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Cockpit 275
+author:
+date: '2022-08-24'
+tags: cockpit, podman
+slug: cockpit-275
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 275 and cockpit-podman 53:
+
+
+## Metrics: Add read/write usage per disk
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit/pull/17657>
+{:.warn}
+
+Adds popover with read/write stats for all disks. Fixes bug where paritions on
+NVMe are not skipped which results in incorrect read/write stats.
+
+![screenshot of add read/write usage per disk]({{ site.baseurl }}/images/275-metrics-add-read-write-usage-per-disk.png)
+
+The button is hidden when only 1 disk is found.
+![screenshot of add read/write usage per disk]({{ site.baseurl }}/images/275-metrics-add-read-write-usage-per-disk-2.png)
+
+On phone:
+![screenshot of add read/write usage per disk]({{ site.baseurl }}/images/275-metrics-add-read-write-usage-per-disk-3.png)
+
+## Shell: Support for alternatives to sudo
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/17536>
+{:.note}
+
+Cockpit always had the ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now the UI lets you select which program to use when multiple are defined in the manifests.
+
+By default, Cockpit will still only use "sudo", so unless you actually add more programs via a manifest, there will be no visible change in the UI.
+
+
+## Image history
```

Comment:
## cockpit-podman: Image history

Reply:
We use `Images` & `Podman` in older release notes so I kept it consistent. Maybe _scripts/generate-release-notes should prefix.